### DISCUSSION TITLE
option to change date format

### DISCUSSION THREAD

**Discussion by Hamisoptimistic** [coddingtonbear/obsidian-web#207] [Q&A] [answered]
Currently the default "Date" returns date: time which i don't like
would love to have where i can insert my preferred Date format.

currently tinkering with typescript code to see something works

**Comment by coddingtonbear** [comment, marked as answer]
You have control over the date format directly; see https://github.com/coddingtonbear/obsidian-web/wiki/Understanding-Templates#helpers.

**Reply by Hamisoptimistic** [reply]
Thanks for this
